flink-connectors/flink-jdbc/src/test/java/org/apache/flink/api/java/io/jdbc/JDBCTestBase.java

```diff
@@ -95,7 +99,7 @@ public static String getCreateQuery(String tableName) {
 		sqlQueryBuilder.append("id INT NOT NULL DEFAULT 0,");
 		sqlQueryBuilder.append("title VARCHAR(50) DEFAULT NULL,");
 		sqlQueryBuilder.append("author VARCHAR(50) DEFAULT NULL,");
-		sqlQueryBuilder.append("price FLOAT DEFAULT NULL,");
+		sqlQueryBuilder.append("price DOUBLE DEFAULT NULL,");
```

Comment:
Why need to change to DOUBLE?

Reply:
The type is `Double` in `TestEntry`. But as double and float are the same in derby, maybe I should change it back.